_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
 
 # Tarball
 
-The tarball installation provides a self-contained directory with everything you need to run OpenSearch, including an integrated Java Development Kit (JDK). The tarball is a good option for testing and development.
+Installing OpenSearch from a tarball, also known as a tar archive, might appeal to users who want granular control over installation details like file permissions and installation paths.
 
-The tarball supports most Linux distributions, including CentOS 7, Amazon Linux 2, and Ubuntu 18.04. If you have your own Java installation and set `JAVA_HOME` in the terminal, macOS works, as well.
+Generally speaking, the installation of OpenSearch from a tarball can be broken down in to a few steps:
 
-1. Download the tarball from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.
+1. **Download and Unpack OpenSearch**
```

Comment:
Should steps be sentence case? And the corresponding headings?

Reply:
I'm not sure.  Heather reviewed that section and said it looked good.  This is heading to Nathan for editorial review next so I'll get feedback from him.

Reply:
Yes, steps should be sentence case.